package.json

```diff
@@ -243,7 +242,14 @@
     "ts-mocha": "^10.0.0",
     "tsd": "^0.27.0",
     "typescript": "^5.0.2",
-    "url": "^0.11.0"
+    "url": "^0.11.0",
+
+    "@docusaurus/core": "^2.0.0",
+    "@easyops-cn/docusaurus-search-local": "^0.33.6",
+    "@mdx-js/react": "^1.6.21",
+    "clsx": "^1.1.1",
+    "docusaurus-preset-openapi": "0.6.4",
+    "prism-react-renderer": "^1.2.1"
```

Comment:
I've not done it yet, but one of the bits of this I am looking forward to most is really going to town deleting all the dependencies for gatsby, typescript, etc. I'm going to leave this until the end though just to minimise the need to solve conflicts. Lets get to a stage where we're happy with the code changes first. Then I'll do that and clean this up. I also need to go through the prod dependencies - there's some stuff like `@fontsource/lato` that wants deleting from there too.